bower.json

```diff
@@ -15,6 +15,7 @@
         "purescript-newtype": "^3.0.0"
     },
     "devDependencies": {
+        "purescript-psci-support": "^4.0.0",
```

Comment:
I also added this so that `pulp repl` can easily be used.  I can remove this if you don't want it.